azure-functions-powershell-library/Common/Constants.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
```

Comment:
Could you please add the license to the top of the file?

```
//
// Copyright (c) Microsoft. All rights reserved.
// Licensed under the MIT license. See LICENSE file in the project root for full license information.
//
```

Reply:
Done